ledger/test-common/src/main/daml/model/Foo.daml

```diff
@@ -12,6 +12,20 @@ template Foo1
     signatory signatory
     observer observers
 
```

Comment:
Would it make sense to have a daml for the api bench tool separately?
Does someone else uses Foo?

Reply:
Discussed offline.

For the record:
Foo.daml had been introduced specifically for the BenchTool. We might change its name and location in a future improvement.